### ISSUE TITLE
Document / sunset Heroku usage

### ISSUE THREAD

**Issue by rianfowler** [department-of-veterans-affairs/va.gov-team#2522]
# Overview 

- Heroku instances are used for content edit reviews 
- with the migration of the CMS to drupal, content editors should be using the the preview server / drupal to review content changes 

# Tasks 

- Evaluate the need for heroku instances (e.g. are people using them, what for)
- Determine a plan for sunsetting heroku build process / review instances

**Comment by wyattwalter** [comment]
Note that review instances are also used by API devs as well, they contain both front- and back-end components together.

**Comment by wyattwalter** [comment]
@rianfowler I discovered that the Heroku builds were connected to GitHub by my user by mistake for vagov-content. I corrected that today, and discovered that they've been broken for some time anyway. The build is trying to connect to the CMS which doesn't work off the VA network. We could probably just sunset the vagov-content stuff without much notice at this point. I can't tell how long they've been broken, but if you're not hearing complaints about it, then there's probably not much of a point in putting more work in here.
